views/js/controller/resultsMonitoring.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2014-2018 (original work) Open Assessment Technologies SA;
+ */
+/**
+ * @author Aleksej Tikhanovich, <aleksej@taotesting.com>
+ */
+
+define([
+    'jquery',
+    'i18n',
+    'util/url',
+    'uri',
+    'ui/feedback',
+    'util/locale',
+    'util/encode',
+    'layout/loading-bar',
+    'layout/actions/binder',
+    'ui/dialog/confirm',
+    'ui/datatable'
+], function ($, __, url, uri, feedback, locale, encode, loadingBar, binder, dialogConfirm) {
+    'use strict';
+
+    var $resultsListContainer = $('.results-list-container');
+
+    function getRequestErrorMessage (xhr) {
+        loadingBar.start();
+        var message = '';
+        try {
+            var responseJSON = $.parseJSON(xhr.responseText);
+            if (responseJSON.message) {
+                message = responseJSON.message;
+            } else {
+                message = xhr.responseText;
+            }
+        } catch (e) {
+            message = xhr.responseText;
+        }
+        return message;
+    }
+
+    function viewResult(rowId) {
+        var res = parseRowId(rowId);
+        loadingBar.start();
+        $.ajax({
+            url : url.route('viewResult', 'Results', 'taoOutcomeUi', {id : res[0], classUri: res[1]}),
+            type : 'GET',
+            success : function (result) {
+                var $container = $('<div class="modal results-modal"></div>').append(result);
+                $resultsListContainer.append($container);
+                $container.modal({
+                    startClosed : true,
+                    minWidth : 450
+                });
+                $container.modal('open');
```

Comment:
For what purpose this modal is intended for?
If it is to display a message, please consider to use `dialogAlert()`.
You may also take a look at `dialog()`, that provides common way to display dialog boxes.
Otherwise, please always use a template instead of hardcoded HTML injection.